project1/css/index.css

```diff
@@ -54,11 +54,92 @@ table {
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
```

Comment:
Try to stay consistent with indentation. Pick either spaces or tabs and stick to one!